test/runtests.jl

```diff
@@ -106,6 +106,14 @@ module wrapper12
 	end
 end
 
+module wrapper13
+	using FromFile
+	visible = [:D]
+	invisible = [:foo, :bar, :baz, :quux, :A, :B, :C]
+
+	@from "basic.jl" import A as D
+end
```

Comment:
we need to be compatible with old julia

```suggestion
if VERSION ≥ v"1.6"
module wrapper13
	using FromFile
	visible = [:D]
	invisible = [:foo, :bar, :baz, :quux, :A, :B, :C]

	@from "basic.jl" import A as D
end

end # VERSION ≥ v"1.6"
```